src/main/java/com/sogeti/SoSenseiPsyche/model/Game.java

```diff
@@ -1,26 +1,32 @@
 package com.sogeti.SoSenseiPsyche.model;
 
 import com.sogeti.SoSenseiPsyche.helpers.ColorMapper;
+import com.sogeti.SoSenseiPsyche.record.FeedbackRecord;
 import com.sogeti.SoSenseiPsyche.validations.GuessValidation;
 
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
 import java.util.Scanner;
 
+import static com.sogeti.SoSenseiPsyche.model.GameboardCreator.printGameBoard;
+
 
 public class Game {
     public static final int CODE_LENGTH = 4;
+    private int attempt = 1;
     private int attemptsRemaining = 12;
-    private List<String> guesses;
     private List<Character> secretCode;
     private final Scanner scanner = new Scanner(System.in);
+    private List<FeedbackRecord> feedbackList;
+    private final Feedback feedback = new Feedback();
+    GuessValidation guessValidation = new GuessValidation();
 
     public void startGame() {
         Code code = new Code();
-        guesses = new ArrayList<>();
+//        guesses = new ArrayList<>();
```

Comment:
probably the commented line should be removed

Reply:
removed